src/CONST.ts

```diff
@@ -2283,6 +2283,7 @@ const CONST = {
         PRIVATE_NOTES: 'privateNotes',
         DELETE: 'delete',
         MARK_AS_INCOMPLETE: 'markAsIncomplete',
+        BOOK_A_CALL: 'bookACall',
```

Comment:
Where is this used?

Reply:
This was a leftover from when Book a call was still in ReportDetailsPage, I've moved it to ProfilePage since so this can be removed, good catch.